test/validation/spec.postcode-validation.js

```diff
@@ -20,8 +20,8 @@ describe('Postcode validation', () => {
     });
 
     describeUnlessCoverage('Full postcode test - loads full UK postcode database, may take some time', function () {
-        this.timeout(40000);
-        this.slow(20000);
+        this.timeout(60000);
```

Comment:
Do we want to have a separate MR for removing the postcode validation? We don't use it anymore, and increasing the timeouts is now getting silly.